public/index.html

```diff
@@ -18,8 +21,38 @@
       ga('create', 'UA-36512724-2', 'auto');
       ga('send', 'pageview');
     </script>
+
+    <style>
+
+      html {
+        box-sizing: border-box; /* border-box */
```

Comment:
Yes, this is to specify what is needed in a global reset that should be included in `tachyons-egghead`; you can't put a `class='border-box'` on an html node for example and some things should cascade with CSS, like the default font families and body background color etc. We should talk about this and get this merged into `tachyons-egghad` with the reset stuff @DrShpongle was doing.